bluesky/run_engine.py

```diff
@@ -748,6 +748,8 @@ def _open_run(self, msg):
 
         # Metadata can come from history, __call__, or the open_run Msg.
         self._metadata_per_run = dict(self.md)
```

Comment:
This is my mistake, not yours, but can we change this to `.update()` as well for philosophical consistency?